lib/config/options/index.ts

```diff
@@ -1955,25 +1954,6 @@ const options: RenovateOptions[] = [
     cli: false,
     env: false,
   },
-  {
-    name: 'js',
-    description: 'Configuration object for JavaScript language.',
-    stage: 'package',
-    type: 'object',
-    default: {},
-    mergeable: true,
-  },
-  {
-    name: 'golang',
-    description: 'Configuration object for Go language.',
-    stage: 'package',
-    type: 'object',
-    default: {
-      commitMessageTopic: 'module {{depName}}',
```

Comment:
this will cause commit and PR name changes.

@rarkins should we somehow create legacy presets to retain those default settings which are removed now?

Reply:
Although the way of configuring it is legacy, we probably want to preserve the functionality like this in most cases? Maybe we can apply `module {{depName}}` to the go datasource instead, and preserve most of it?